CHANGELOG.md

```diff
@@ -62,6 +62,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Minor refactor of `@cumulus/lzards-api-client` to:
     - Use proper ECMAScript import for `@cumulus/launchpad-auth`
     - Update incorrect docstring
+- **CUMULUS-3449**
+  - Updated `@cumulus/db` package and add knex hook postProcessResponse to convert postgres id fields in
```

Comment:
This looks MUCH cleaner @jennyhliu  than version 2, my only concern is that we have a conversion by db schema naming convention (same as #2) and thinking about this more it might be a good idea to do something like 

```
if BigInt(Number.MAX_SAFE_INTEGER) <= BigInt(Number(value)) { throw } 
```

if the impact to performance isn't serious.